terraform/ipfs-mirror/nginx-default

```diff
@@ -42,9 +42,17 @@ server {
     # First attempt to serve request as file, then
     # as directory, then fall back to displaying a 404.
     try_files $uri $uri/ =404;
+    
+    # Prevent Caching
+    expires           0;
+    add_header        Cache-Control private;
```

Comment:
I think `no-cache` is better?

https://www.w3.org/Protocols/HTTP/Issues/cache-private.html

Reply:
Any value adding `no-store` and `must-revalidate` in addition to this?
https://stackoverflow.com/questions/11357430/http-expires-header-values-0-and-1